example.env

```diff
@@ -1 +1 @@
-REPORT_FILE_REPOSITORY_PATH=report_files_repository
\ No newline at end of file
+REPORT_FILE_REPOSITORY_DIR=report_files_repository
```

Comment:
It would be good to include a comment with an example of `report_files_repository` value

Reply:
I changed it from `REPORT_FILE_REPOSITORY_DIR` to `REPORT_FILE_DIR` and I think it's clearer.